packages/trader/src/Constants/contract.js

```diff
@@ -34,64 +34,64 @@ export const getCardLabels = () => ({
-    R_75: <Localize i18n_default_text='Volatility 75 Index' />,
-    R_100: <Localize i18n_default_text='Volatility 100 Index' />,
-    RDBEAR: <Localize i18n_default_text='Bear Market Index' />,
-    RDBULL: <Localize i18n_default_text='Bull Market Index' />,
-    WLDAUD: <Localize i18n_default_text='AUD Index' />,
-    WLDEUR: <Localize i18n_default_text='EUR Index' />,
-    WLDGBP: <Localize i18n_default_text='GBP Index' />,
-    WLDUSD: <Localize i18n_default_text='USD Index' />,
-    '1HZ10V': <Localize i18n_default_text='Volatility 10 (1s) Index' />,
-    '1HZ100V': <Localize i18n_default_text='Volatility 100 (1s) Index' />,
+    FRXAUDCAD: localize('AUD/CAD'),
+    FRXAUDCHF: localize('AUD/CHF'),
+    FRXAUDJPY: localize('AUD/JPY'),
+    FRXAUDNZD: localize('AUD/NZD'),
+    FRXAUDPLN: localize('AUD/PLN'),
+    FRXAUDUSD: localize('AUD/USD'),
+    FRXBROUSD: localize('Oil/USD'),
+    FRXEURAUD: localize('EUR/AUD'),
+    FRXEURCAD: localize('EUR/CAD'),
+    FRXEURCHF: localize('EUR/CHF'),
+    FRXEURGBP: localize('EUR/GBP'),
+    FRXEURJPY: localize('EUR/JPY'),
+    FRXEURNZD: localize('EUR/NZD'),
+    FRXEURUSD: localize('EUR/USD'),
+    FRXGBPAUD: localize('GBP/AUD'),
+    FRXGBPCAD: localize('GBP/CAD'),
+    FRXGBPCHF: localize('GBP/CHF'),
+    FRXGBPJPY: localize('GBP/JPY'),
+    FRXGBPNOK: localize('GBP/NOK'),
+    FRXGBPUSD: localize('GBP/USD'),
+    FRXNZDJPY: localize('NZD/JPY'),
+    FRXNZDUSD: localize('NZD/USD'),
+    FRXUSDCAD: localize('USD/CAD'),
+    FRXUSDCHF: localize('USD/CHF'),
+    FRXUSDJPY: localize('USD/JPY'),
+    FRXUSDNOK: localize('USD/NOK'),
+    FRXUSDPLN: localize('USD/PLN'),
+    FRXUSDSEK: localize('USD/SEK'),
+    FRXXAGUSD: localize('Silver/USD'),
+    FRXXAUUSD: localize('Gold/USD'),
+    FRXXPDUSD: localize('Palladium/USD'),
+    FRXXPTUSD: localize('Platinum/USD'),
+    OTC_AEX: localize('Dutch Index'),
+    OTC_AS51: localize('Australian Index'),
+    OTC_DJI: localize('Wall Street Index'),
+    OTC_FCHI: localize('French Index'),
+    OTC_FTSE: localize('UK Index'),
+    OTC_GDAXI: localize('German Index'),
+    OTC_HSI: localize('Hong Kong Index'),
+    OTC_IBEX35: localize('Spanish Index'),
+    OTC_N225: localize('Japanese Index'),
+    OTC_NDX: localize('US Tech Index'),
+    OTC_SPC: localize('US Index'),
+    OTC_SSMI: localize('Swiss Index'),
+    OTC_SX5E: localize('Euro 50 Index'),
+    R_10: localize('Volatility 10 Index'),
+    R_25: localize('Volatility 25 Index'),
+    R_50: localize('Volatility 50 Index'),
+    R_75: localize('Volatility 75 Index'),
+    R_100: localize('Volatility 100 Index'),
+    RDBEAR: localize('Bear Market Index'),
+    RDBULL: localize('Bull Market Index'),
+    WLDAUD: localize('AUD Index'),
+    WLDEUR: localize('EUR Index'),
+    WLDGBP: localize('GBP Index'),
+    WLDUSD: localize('USD Index'),
+    '1HZ10V': localize('Volatility 10 (1s) Index'),
+    '1HZ100V': localize('Volatility 100 (1s) Index'),
```

Comment:
What's the reason for substituting `<Localize />` with `localize`?

Reply:
Because popover msg in digits will fail to display the correct msg and it displays [object object].
After the change, translation works as good as before.